container-core/pom.xml

```diff
@@ -453,6 +453,8 @@
             <arg>-Xlint:-try</arg>
             <arg>-Werror</arg>
           </compilerArgs>
+            <source>17</source>
```

Comment:
Why is this needed?

Reply:
Sorry, it's not, just more IntelliJ inserts - deleted.